src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/service/JJDisputeService.java

```diff
@@ -61,28 +54,52 @@ public JJDispute updateJJDispute(String id, JJDispute jjDispute) {
 		JJDisputeStatus jjDisputeStatus = jjDispute.getStatus();
 		
 		// TCVP-1435 - business rules
-		// - current status must be NEW, REVIEW, IN_PROGRESS to change to IN_PROGRESS
-		// - current status must be NEW, IN_PROGRESS, REVIEW to change to REVIEW
-		// - current status must be REVIEW to change to COMPLETED
+		// - current status must be NEW, IN_PROGRESS to change to IN_PROGRESS
+		// - current status must be CONFIRMED, REVIEW to change to REVIEW
+		// - current status must be NEW, IN_PROGRESS, REVIEW, CONFIRMED to change to CONFIRMED
+		// - current status must be REVIEW, NEW, IN_PROGRESS to change to CONFIRMED
```

Comment:
I think the second rule definition to change to CONFIRMED is redundant since the logic below follows the first definition.